docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx

```diff
@@ -72,35 +72,34 @@ npx suspensive-react-query --help
 
 ### status
 
-Checks and displays the interface compatibility of @suspensive/react-query with the currently used version of @tanstack/react-query. It also lists the available APIs based on the version.
+Checks the compatibility of the currently installed version of @tanstack/react-query with @suspensive/react-query and displays the compatibility status.
 
 ```bash
 npx suspensive-react-query status
 ```
 
-For instance, if using @suspensive/react-query@2.9.0 and @tanstack/react-query@5, the result might look like this:
+For example, using @suspensive/react-query@2.12.1 and @tanstack/react-query@5.51.21, the result might be.
+
+You can check the compatibility status with status, and see which interfaces are available in your environment in the available interfaces section.
```

Comment:
```suggestion
You can check the compatibility status with `npx suspensive-react-query status`, and see which interfaces are available in your environment in the available interfaces section.
```

Reply:
I noticed some duplicate content, so I made slight revisions to that section.

```md
```